snorkel/contrib/fonduer/phantomjs/examples/arguments.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
Remove phantomjs. We have the install script, so this whole program shouldn't be versioned in this repo.